src/index.html

```diff
@@ -6,8 +6,28 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <title>Product cards</title>
   <link rel="stylesheet" href="./style.css">
+  <link rel="icon" href="./images/logo.png">
 </head>
 <body>
-  <h1>Product cards</h1>
+  <article class="product-card" data-qa="card">
+    <img class="product-card__img" src="./images/imac.jpeg" alt="product image"></img>
+    <span class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</span>
```

Comment:
wrap it in some header tag

Reply:
added classes "product-card__header" and  "product-card__content"